app/controllers/admin/backups_controller.rb

```diff
@@ -72,7 +72,7 @@ def email
   def show
     if !EmailBackupToken.compare(current_user.id, params.fetch(:token))
       @error = I18n.t('download_backup_mailer.no_token')
-      return render template: 'admin/backups/show.html.erb', layout: 'no_ember', status: 422
+      return render layout: 'no_ember', status: 422, formats: [:html]
```

Comment:
Does this mean `app/views/admin/backups/show.html.erb` can be deleted?

Reply:
No it's still needed; Rails infers the template path from both the controller name and action name and it automatically renders `admin/backups/show` for us.